connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -2442,10 +2442,18 @@ function getTableInfoFromDatabase(database: NotionDatabase): {
   tableDescription: string;
 } {
   const tableId = getNotionDatabaseTableId(database.notionDatabaseId);
-  const name =
-    database.title ?? `Untitled Database (${database.notionDatabaseId})`;
-  const tableName = slugify(name.substring(0, 32));
+  const fallbackName = `Untitled Database (${database.notionDatabaseId})`;
+
+  let tableName = slugify((database.title ?? "").substring(0, 32));
+  if (!tableName) {
```

Comment:
`slugify` always returns a string, so worst case `tableName` will be an empty string, no?

Reply:
ok but then why not:
```
let tableName = slugify((database.title ||  fallbackName).substring(0, 32));
```

Reply:
Because you have no guarantee that slugify does not replace some characters by `""`

Reply:
Today is not the case but in the future you don't know